### ISSUE TITLE
Bug of writing uint8_t as a character instead of integer value in final config

### ISSUE THREAD

**Issue by aka2en** [sparcians/map#342]
I declared a Sparta parameter by uint8_t type, then Sparta wrote a character instead of integer value in final config file. That caused Sparta failed to read the same parameter from the final config file, because it was finding a uint8_t value.

**Comment by knute-sifive** [comment]
Yep, can reproduce.  Got a test on branch `knutel/issue_342_int8_params`

**Comment by knute-sifive** [comment]
Hey @furuame, is this still an issue?  Did you try the branch?
